### PR TITLE
Change all occurrences of private address to zkAddress

### LINKED ISSUE #87: Change all occurrences of private address to zkAddress
To be more consistent with company wide language.

### PR DESCRIPTION
**Pull request by ghzlatarev**
closes #87

* Change all occurences of privateAddress to zkAddress to keep it consistent with the company's language 
* Fix spelling mistake in `initalWlaletSync` -> `initialWalletSync`

---

Before we can merge this PR, please make sure that all the following items have been checked off:

- [x] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/sdk/blob/main/CHANGELOG.md) and added the appropriate `L-` label to the PR.
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.

### CODE REVIEW

**Comment by DanielZhangReal** [comment]
Does the brand need to emphasize privateAddress instead of zkAddress?

The SDK will have relatively large changes after the development of the Extension Wallet is completed, so is it possible to consider changing these at that time?

**Review by DanielZhangReal — DISMISSED** [review]

File: manta-js/package/src/privateWallet.ts [hunk 1]
```diff
@@ -94,21 +94,21 @@ export class MantaPrivateWallet implements IMantaPrivateWallet {
     return config.NETWORKS;
   }
 
-  /// Returns the ZkAddress (Private Address) of the currently connected manta-signer instance.
+  /// Returns the ZkAddress (ZK Address) of the currently connected manta-signer instance.
```
Comment (ferrell-code) [inline manta-js/package/src/privateWallet.ts:97]:
```suggestion
  /// Returns the zkAddress of the currently connected manta-signer instance.
```

Comment (ferrell-code) [reply manta-js/package/src/privateWallet.ts:97]: replace pasta

Comment (ghzlatarev) [reply manta-js/package/src/privateWallet.ts]: thx

**Review by ferrell-code — APPROVED** [review]

**Review by ferrell-code — APPROVED** [review]

**Review by DanielZhangReal — APPROVED** [review]

**Review by DanielZhangReal — APPROVED** [review]

**Pull request merged**